src/utilities/History/add-history.js

```diff
@@ -0,0 +1,24 @@
+import axios from "axios";
+
+export const addToHistory = async (video, dispatchVideo) => {
+    const token = localStorage.getItem("token");
```

Comment:
consider edge cases if a user is not signed in when adding to history.

Reply:
Thanks for the suggestion, will do it while refactoring it later.